Small little amendments to the topics within the conceptual section … (#7)
postsharp/Metalama.Documentation: conceptual/Aspects/ide/code-fixes.md

## Attaching code fixes to diagnostics

> Whenever an aspect or fabric reports a diagnostic, it can attach <!--- to What? --> as set of code fixes by calling the <xref:Metalama.Framework.Diagnostics.IDiagnostic.WithCodeFixes*?text=IDiagnostic.WithCodeFixes> method. To create one-step code fixes, you can use the <xref:Metalama.Framework.CodeFixes.CodeFixFactory> class.

Reviewer: ```suggestion
Whenever an aspect or fabric reports a diagnostic, it can attach a set of code fixes to this diagnostic by calling the <xref:Metalama.Framework.Diagnostics.IDiagnostic.WithCodeFixes*?text=IDiagnostic.WithCodeFixes> method. To create one-step code fixes, you can use the <xref:Metalama.Framework.CodeFixes.CodeFixFactory> class.
```
